recipes/r-fastcpd/meta.yaml

```
@@ -0,0 +1,96 @@
+{% set version = "0.7.0" %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-fastcpd
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/fastcpd_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/fastcpd/fastcpd_{{ version }}.tar.gz
+    # - https://github.com/doccstat/fastcpd/releases/download/v{{ version }}/fastcpd-v{{ version }}.tar.gz
+    # - https://github.com/doccstat/fastcpd/archive/v{{ version }}.tar.gz
+    # - https://github.com/doccstat/fastcpd/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 1af082e823686e8e4e2c530054fcaca43711850a493d09dbafefc143c6b13d1c
+
+build:
+  merge_build_host: true  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  run_exports:
+    - r-desctools
+    - r-glmnet
+    - r-matrix
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-testthat
+  skip: true  # [win]
+
+requirements:
+  build:
+    - cross-r-base {{ r_base }}  # [build_platform != target_platform]
+    - {{ compiler('c') }}  # [not win]
+    - {{ compiler('m2w64_c') }}  # [win]
+    - {{ compiler('cxx') }}  # [not win]
+    - {{ compiler('m2w64_cxx') }}  # [win]
+    - {{ posix }}filesystem  # [win]
+    - {{ posix }}make
+    - {{ posix }}sed  # [win]
+    - {{ posix }}coreutils  # [win]
+    - {{ posix }}zip  # [win]
+
+  host:
+    - _openmp_mutex  # [linux]
+    - libgcc-ng  # [linux]
+    - libstdcxx-ng  # [linux]
+    - r-base
+    - r-desctools
+    - r-glmnet
+    - r-matrix
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-testthat
+
+  run:
+    - libopenblas
+    - llvm-openmp  # [osx]
+    - r-base
+    - r-desctools
+    - r-glmnet
+    - r-matrix
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-testthat
+    - {{ native }}gcc-libs  # [win]
+
+test:
+  commands:
+    - $R -e "library('fastcpd')"  # [not win]
+    - "\"%R%\" -e \"library('fastcpd')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=fastcpd, https://github.com/doccstat/fastcpd
+  summary: Fast Change Point Detection via Sequential Gradient Descent
+  description: |
+    Implements fast change point detection algorithm based on the paper
+    "Sequential Gradient Descent and Quasi-Newton's Method for Change-Point Analysis"
+    by Xianyang Zhang, Trisha Dawn <https://proceedings.mlr.press/v206/zhang23b.html>.
+    The algorithm is based on dynamic programming with pruning and sequential gradient
+    descent. It is able to detect change points a magnitude faster than the vanilla
+    Pruned Exact Linear Time(PELT). The package includes examples of linear
+    regression, logistic regression, Poisson regression, penalized linear regression
+    data, and whole lot more examples with custom cost function in case the user
+    wants to use their own cost function.
+  license: GPL-2.0-or-later
```

DESCRIPTION file shows `GPL (>= 3)`, so...

```suggestion
  license: GPL-3.0-or-later
```